src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -723,4 +723,4 @@ public ProcessDataReceivedEventArgs(string data)
 
         public string Data { get; private set; }
     }
-}
+}
```

Comment:
there was one extra line (blank line) at the end, which is getting removed now. Do you still want me to revert?

---
In reply to: [118039558](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118039558) [](ancestors = 118039558)